pkg/syncutil/syncannotationreader.go

```diff
@@ -0,0 +1,87 @@
+package syncutil
```

Comment:
could we put this in its own package under `syncutil`? something like `parser`?

Reply:
Makes sense! Done